### PR TITLE
CBG-3108 only log relevant SG indexes

### PR DESCRIPTION
**Pull request by torcolvin**
In the first commit, just log SG only indexes. I also add which indexes _aren't_ ready yet so we know which indexes are slow to create.

I think the second commit is more comprehensive about logging but might be too much for 3.1.1

## Pre-review checklist
- [x] Removed debug logging (`fmt.Print`, `log.Print`, ...)
- [x] Logging sensitive data? Make sure it's tagged (e.g. `base.UD(docID)`, `base.MD(dbName)`)
- [x] Updated relevant information in the API specifications (such as endpoint descriptions, schemas, ...) in `docs/api`

## [Integration Tests](https://jenkins.sgwdev.com/job/SyncGateway-Integration/build?delay=0sec)
- [ ] `GSI=true,xattrs=true` https://jenkins.sgwdev.com/job/SyncGateway-Integration/000/

### CODE REVIEW

File: rest/config.go [hunk 1]
```diff
@@ -478,6 +478,9 @@ func readFromPath(path string, insecureSkipVerify bool) (rc io.ReadCloser, err e
 
 func (dbConfig *DbConfig) AutoImportEnabled() (bool, error) {
 	if dbConfig.AutoImport == nil {
+		if !dbConfig.UseXattrs() {
```
Comment (adamcfraser) [inline rest/config.go]: Is this change required for this PR?  It seems unrelated.

**Review by adamcfraser — CHANGES_REQUESTED** [review]

File: rest/config.go (hunk 1 above)
Comment (torcolvin) [reply rest/config.go]: This change already exists on master and if I do a `git diff origin/master` I don't see this change so I don't know what github is doing here.

**Review by adamcfraser — APPROVED** [review]
